packages/jest-config/src/index.ts

```diff
@@ -313,9 +307,21 @@ export function readConfigs(
 
         return true;
       })
-      .map((root, projectIndex) =>
-        readConfig(argv, root, true, configPath, projectIndex),
-      );
+      .map((root, projectIndex) => {
+        // Skip on all except: projectIndex === 0, and isGlobalProject, and projects.length === 1
+        const skipArgvConfigOption = !(
+          projectIndex === 0 &&
+          isGlobalProject &&
```

Comment:
Am I missing something, or can we get rid of the whole `isGlobalProject` if we instead asserted that `projects[0] === process.cwd()` (or `=== realpath(process.cwd())` on Windows)?

Reply:
Thanks for the suggestion! Applied your suggestion and got rid of `isGlobalProject` 😁